src/main/java/com/cbms/ui/controller/SystemsController.java

```diff
@@ -111,15 +116,54 @@ public void handle(MouseEvent mouseEvent) {
                 uiUtilities.changeScene(mouseEvent, "/AddSystem");
             }
         });
+
+        //Adding items to the choiceBox (drop down list)
+        sortSystem.getItems().add("Default");
+        sortSystem.getItems().add("Ascending RUL");
+        sortSystem.getItems().add("Descending RUL");
```

Comment:
Can you try to write this and not use hard coded string values? i'm not gonna put a comment on the other lines but i'm talking about every time you reference this. i'm asking because if in the future if you want to add a different language support it would be much more complex like this